forms/account-freemarker/src/main/java/org/keycloak/account/freemarker/FreeMarkerAccountProvider.java

```diff
@@ -197,7 +197,11 @@ public Response createResponse(AccountPages page) {
             String result = freeMarker.processTemplate(attributes, Templates.getTemplate(page), theme);
             Response.ResponseBuilder builder = Response.status(status).type(MediaType.TEXT_HTML).entity(result);
             BrowserSecurityHeaderSetup.headers(builder, realm);
-            LocaleHelper.updateLocaleCookie(builder, locale, realm, uriInfo, Urls.localeCookiePath(baseUri,realm.getName()));
+
+            String keycloakLocaleCookiePath = Urls.localeCookiePath(baseUri, realm.getName());
```

Comment:
Unfortunately, we do need two cookies.  We can't change where ng-translate is looking for its cookie.  For Freemarker, the ng-translate cookie won't be sent for its path.  So there is no way to read it.